contributors/onboarding/onboarding.md

```diff
@@ -0,0 +1,44 @@
+
+### Post-Onboarding Readiness 
+
+Post Onboarding Readiness is an internal measure based on feedback throughout a new team member’s first 90 days at Mattermost. There are multiple milestones throughout the onboarding process where feedback will be solicited. This feedback intends to assess how willing and enabled a new colleague is to bring their best self to Mattermost as it relates to their role expectations, team and company focuses and the Mattermost Leadership Principles. 
+
+Feedback will be requested throughout the 90-day Onboarding These consistent check-ins help new colleagues, HR and managers iterate with impact quickly. For example, one of the [V2MOM - Destination Workplace Measures](https://docs.google.com/presentation/d/1BDSaeW-M92gth_NM1vI23dtbFLBsYVIk/edit#slide=id.g6f2ea8cda4_8_1852) is to have fewer than 5% questions repeat month-over-month. 
+
+#### Onboarding Feedback to assess post-onboarding readiness 
+
+**Onboarding Surveys**
+TBD
+
+**New Staff Feedback Session with CEO and HR**
+Similar to [CEO listening tours](https://handbook.mattermost.com/operations/operations/company-cadence#ceo-listening-tours) new colleagues will be asked to share [likes and wishes](https://handbook.mattermost.com/company/about-mattermost/mindsets#likes-and-wishes) about anything Mattermost and Onboarding. 
+
+**New Colleague Feedback**
+The 90-Day Feedback provides a [360](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared) [shoulder check](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check) for new team members, managers and [HR](https://handbook.mattermost.com/operations/workplace/people#team) and MLT to understand and clarify potential blindspots as well as catch potential [brown m&ms](https://handbook.mattermost.com/company/about-mattermost/mindsets#brown-m-and-ms) early on. 
+
+This is not a formal performance review. The intent is to align expectations and improve performance early on. Peer feedback is not anonymous. Managers may request additional context from the panel to help increase the new colleague's performance and post-onboarding readiness. Managers should evaluate the peer feedback against their own expectations of the new colelague's role. In some cases, expectations may need to be re-set with peers. Uncovering blindspots early on helps avoid future confusion or misfocused efforts. 
+
+**New Colleague 90-Day Feedback Process**
+
+ * **Managers**
+    * Complete a [new colleague peer panel form] for the new staff member. Notifications are sent to managers on a new colleague's date of employmnet. This will kick off the [review process](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared). 
```

Comment:
```suggestion
    * Complete a [new colleague peer panel form] for the new staff member. Notifications are sent to managers on a new colleague's date of employment. This will kick off the [review process](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared). 
```
Missing a link for the new colleague peer panel form.